qe_evidence.tendrl.yml

```diff
@@ -49,13 +70,22 @@
       - tendrl-node-agent
       - tendrl-performance-monitoring
   roles:
-    - role: qe-evidence
+    - qe-evidence
     - qe-evidence-probe-journald
 
-#- name: Download logs from Ceph Storage machines
-#  hosts: ceph
-#  user: root
-#  vars:
-#    evidence_dirs:
-#  roles:
-#   - role: qe-evidence
+- name: Download logs from Ceph Storage machines
+  hosts: ceph
```

Comment:
I would replace `ceph` by `ceph_osd:ceph_mon` to follow default groups used in _usmqe-setup_.